### ISSUE TITLE
debug feedback 

### ISSUE THREAD

**Issue by DecentMakeover** [zylo117/Yet-Another-EfficientDet-Pytorch#155]
Custom dataset eff-0 with weights in debug mode.

After 6 epochs.
![0f19b43182a44ca0b5098f43615d6d0a](https://user-images.githubusercontent.com/25679895/79798873-3a426c80-8377-11ea-8dbf-9c5fead3086d.jpg)

Is this like normal ? or .....

Thanks in advance.

**Comment by zylo117** [comment]
it happens when it's in early stage. But it can also be dataset corruption. Is your dataset using 1-indexed class id? In another word, the first category id should be 1, not 0.

**Comment by DecentMakeover** [comment]
I do have a catergory id which is 0. is that a problem? should i just start it from 1?
kind of like 

label 0 = category_id  1 
label 1 = category_id  2??

**Comment by zylo117** [comment]
yes, category id should start at 1.
can you share the loss graph?

**Comment by DecentMakeover** [comment]
Here you go 
the first one is classification and the second, regression 
![Screenshot from 2020-04-21 14-55-09](https://user-images.githubusercontent.com/25679895/79849473-254df380-83e0-11ea-856b-73e8300c6b9e.png)
![Screenshot from 2020-04-21 14-56-27](https://user-images.githubusercontent.com/25679895/79849602-4e6e8400-83e0-11ea-95d0-42c8122a2148.png)

**Comment by zylo117** [comment]
No wonder!
It's totally overfitting!
The cls loss is off the roof!
Please reduce the lr by 100 times or more. If it doesn't help, your dataset could be corrupted like mislabeling or whatnot.

**Comment by DecentMakeover** [comment]
you mean start with a learning rate of 0.000001 ?

**Comment by zylo117** [comment]
no, I meant reduce it to 1e-6.

But consider you have already train on such small lr, you should check you dataset's annotations. Try to visualize them.

**Comment by DecentMakeover** [comment]
alright , thanks for all the inputs.
